community_components.md

```diff
@@ -42,6 +42,12 @@ Master thesis which consists of a completely automated system that provides
 a web interface for configuring GrimoireLab, so that it analyizes any set of
 repositories. [More information (in Spanish)](http://zhquan.github.io/TFG/).
 
+* [Prosoul](https://github.com/Bitergia/prosoul)
+The goal of this project is to create a web editor and viewer for software quality models that are used to show metrics in a meaningful way. Also, importers and exporters for the different quality models used in the industry will be implemented so they can be used as models or evolved for the creation of new quality models. In the initial version the metrics supported will be the CROSSMINER ones (see below), but it will aso support GrimoireLab data. Based on the models, visualizations and assessment for the projects are generated. In the inital version visualizations will be created using GrimoireLab.
```

Comment:
I think it would be needed to stress the relationship with GrimoireLab. Right now, it is not clear (I assume it is that it uses GrimoireLab data, but in that case, it would be good to know if that´s data directly obtained from Perceval, or raw indexes, or enriched indexes, or what...)

I think the explanation should better not reference an entry for CROSSMiner, but directly a link to CROSSMiner (see below).